pkgs/applications/editors/texstudio/default.nix

```
@@ -29,6 +29,12 @@ stdenv.mkDerivation (finalAttrs: {
     zlib
   ];
 
+  preFixup = lib.optionalString stdenv.isDarwin ''
+    mkdir -p "$out/Applications"
+    OUT_APP="$out/Applications/texstudio.app"
+    cp -r "$out/bin/texstudio.app" "$OUT_APP"
```

Why not mv? Also, that variable seems pretty unnecessary.

```suggestion
    mv "$out/bin/texstudio.app" "$out/Applications"
```